data/crac-creation/crac-creator-csa-profiles/src/main/java/com/farao_community/farao/data/crac_creation/creator/csa_profile/CsaProfileCrac.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.farao_community.farao.data.crac_creation.creator.csa_profile;
+
+import com.farao_community.farao.data.crac_creation.creator.csa_profile.crac_creator.CsaProfileConstants;
+import com.farao_community.farao.data.native_crac_api.NativeCrac;
+import com.powsybl.triplestore.api.PropertyBags;
+import com.powsybl.triplestore.api.QueryCatalog;
+import com.powsybl.triplestore.api.TripleStore;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * @author Jean-Pierre Arnould {@literal <jean-pierre.arnould at rte-france.com>}
+ */
+public class CsaProfileCrac implements NativeCrac {
+
+    private final TripleStore tripleStoreCsaProfileCrac;
+
+    private final QueryCatalog queryCatalogCsaProfileCrac;
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(CsaProfileCrac.class);
```

Comment:
we mostly use FaraoLoggerProvider for logs in FARAO and the ulitmate goal is to stop using residual LOGGER.